docs/content/concepts/sui-architecture/networks.mdx

```diff
@@ -1,6 +1,51 @@
+
+Validator nodes can sign transactions but only store the latest transactions.
+
+Learn more about [nodes on Sui](/guides/operator.mdx).
+
+## Mainnet
+
+Mainnet is responsible for all production environment applications and transactions. All activity on Mainnet costs real SUI or MIST tokens, which have a value of fiat currency. SUI or MIST must be bought from an exchange, and excess SUI or MIST can be sold at a later date on the exchange for its fiat value. All activity on Mainnet is public and can be viewed through explorers such as [Sui Explorer](https://suiscan.xyz/mainnet/home).
+
+[Epochs](/concepts/sui-architecture/epochs.mdx) on Mainnet are approximately 24 hours long. Data is always persisted on the network. Learn more about [data storage and storage fees](/concepts/sui-architecture/sui-storage.mdx).
+
+The RPC URL for Mainnet is: `https://fullnode.mainnet.sui.io:443`.
+
+## Testnet
+
+Testnet is a staging network designed for developers to test their code before placing it in production. Activity on Testnet uses Testnet SUI and MIST tokens, which can be obtained through a [faucet](/guides/developer/getting-started/get-coins.mdx). Testnet tokens have no real value and cannot be exchanged for fiat currency.
+
+All activity on Testnet is public and can be viewed through explorers such as [Sui Explorer](https://suiscan.xyz/testnet/home).
+
+[Epochs](/concepts/sui-architecture/epochs.mdx) on Testnet are approximately 24 hours long. Data on Testnet is not guaranteed to be persisted because data might be wiped occasionally. It should not be relied on for permanent storage. Testnet data persists through the regular update process, and planned data wipes are announced ahead of time.
+
+The RPC URL for Testnet is: `https://fullnode.testnet.sui.io:443`.
+
+## Devnet
+
+Devnet is an unstable network used to develop new features. If you wish to write code against upcoming Sui features, you can use this network for early testing before the feature is finalized and released.
+
+All activity on Devnet is public and can be viewed through explorers such as [Sui Explorer](https://suiscan.xyz/devnet/home).
+
+[Epochs](/concepts/sui-architecture/epochs.mdx) on Devnet are 1 hour long. Devnet is not recommended for new developers because it is better suited for advanced use cases. Data on Devnet is wiped **weekly** as part of its [scheduled update process](https://sui.io/networkinfo).
+
+The RPC URL for Devnet is: `https://fullnode.devnet.sui.io:443`.
+
+## Localnet
+
+Localnet is used to run an instance of the Sui network on your local computer. Developing on a local network provides an optimized workflow in an environment you control. You can set custom network configuration parameters, such as data persistence and a local faucet.
```

Comment:
I think the local faucet point could be expanded to help people understand they can get as many tokens as needed.